opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -81,6 +99,57 @@ namespace Opm {
                 for (size_t globalIndex = 0; globalIndex < multpvData.size(); globalIndex++)
                     values[globalIndex] *= multpvData[globalIndex];
             }
+
+            // deal with the region multiplier for porosity
+            if (deck->hasKeyword("MULTREGP")) {
+                const DeckKeyword& multregpKeyword = deck->getKeyword("MULTREGP");
+                for (int recordIdx = 0; recordIdx < multregpKeyword.size(); ++recordIdx) {
+                    const DeckRecord& multregpRecord = multregpKeyword.getRecord(recordIdx);
+
+                    int regionIdx = multregpRecord.getItem("REGION").template get<int>(0);
+                    std::string regionType = multregpRecord.getItem("REGION_TYPE").template get<std::string>(0);
+                    double multValue = multregpRecord.getItem("MULTIPLIER").template get<double>(0);
+                    boost::to_upper(regionType);
```

Comment:
> that's news to me: it was not around too long ago. anyway, it's changed.

Yeah, it's rather recent. It was added during a boost purge where most of these (really only uppercase) boost dependencies were removed, in an effort to improve performance and reduce redundant work.

> yeah, true, but do we want to eat more than eclipse does? (i.e., is it okay if decks get accepted by opm-parser but not by Eclipse?)

I suppose, and it's not the first time we're doing that I think, but it provides us with another dilemma when people start using undocumented accidental behaviour and we want to change something.